Server/datalayer/mongo.datalayer.js

```diff
@@ -17,5 +17,6 @@ class MongoDataLayer {
     static disconnect(){
         return mongoose.connection.close();
     }
+
```

Comment:
please revert this. unintentional change

Reply:
Removed